pyproject.toml

```diff
@@ -65,14 +65,6 @@ python_files = ["*test_*.py"]
 testpaths = ["tests"]
 filterwarnings = "error"
 
```

Comment:
Is the `isort` config no longer neede?

Reply:
I don't think ruff is using it, it's compatible with black by default.

Reply:
But do we get import sorting still then?

Reply:
We might not have intra group sorting with first / third parties, which is why isort is kept in pylint. I think in astroid we'll have "good enough sorting" because we don't have a lot of third parties -- and faster commits.

Reply:
Nevermind, I just tested, ruff sort correctly first party and third party without any configuration.

Reply:
We might want to use ruff instead of isort for wrong-import-order in pylint.